.github/workflows/publish_draft_releases.yml

```diff
@@ -236,8 +236,8 @@ jobs:
       - if: ${{ needs.check-for-runtime-upgrade.outputs.do-versions-match == 'false' }}
```

Comment:
please add a description in the PR - what changed and why.
This helps people who need to read the history of the code.